off_nitrogens/tests/test_improper.py

```diff
@@ -1,4 +1,5 @@
-from off_nitrogens.calc_improper import *
+#from off_nitrogens.calc_improper import *
+from calc_improper import *
```

Comment:
Does this mean you got `from calc_improper import *` it to work?

Reply:
No, this was left over from testing on local. Jessica and I made some changes, and she'll change it back before final commit